cnf-certification-test/accesscontrol/suite.go

```diff
@@ -691,11 +692,23 @@ func testNoSSHDaemonsAllowed(env *provider.TestEnvironment) {
 	var badPods []string
 	var errPods []string
 
-	sshPortInfo := netutil.PortInfo{PortNumber: sshServicePortNumber, Protocol: sshServicePortProtocol}
-
 	for _, put := range env.Pods {
 		cut := put.Containers[0]
 
+		port, err := netutil.GetSSHDaemonPort(cut)
+		if err != nil { // This is fine, as it may be that SSH is not running, hence no sshd_config found
```

Comment:
Would this be a `compliantObject` then?

Reply:
This is not non-compliant object. I would not call it compliantObject as we are finding only the non-compliant ones.